packages/nuxt/src/app/entry.ts

```diff
@@ -75,7 +75,7 @@ if (process.client) {
     try {
       await nuxt.hooks.callHook('app:created', vueApp)
       await nuxt.hooks.callHook('app:beforeMount', vueApp)
-      vueApp.mount('#' + appRootId)
+      vueApp.mount(appRootId ? `#${appRootId}` : `body > ${appRootTag}`)
```

Comment:
it would be nice not to include this code in every app for this edge case. maybe we could use a flag to ensure no extra code is added in default case?